src/components/WhatIsNew/WhatIsNew.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import HttpService from '../../service/HttpService/httpService';
+import ItemInfo from './ItemInfo/ItemInfo';
+import './WhatIsNew.scss';
+
+class WhatIsNew extends Component {
+  state = {
+    items: [],
+  }
+
+  async test() {
```

Comment:
Maybe this function should have more clear title so that it would be understandable what does it load?